src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -541,7 +541,7 @@ export default {
           filingType,
           title: this.filingTypeToName(filingType, agmYear),
           filingId: header.filingId,
-          filingAuthor: header.certifiedBy,
```

Comment:
Note that we were missing the Registry Staff fallback for some of our filings. (Oops.) Fixed now.

Note also that the new behaviour is the same as the previous behaviour (except for the fixed filing types), and is backward compatible with current filings (that don't have the `isStaff` header property).